modules/apps/change-tracking/change-tracking-api/src/main/java/com/liferay/change/tracking/configuration/CTSettingsConfiguration.java

```diff
@@ -29,6 +29,9 @@ public interface CTSettingsConfiguration {
 	)
 	public long defaultCTCollectionTemplateId();
 
+	@Meta.AD(name = "default-owner-action-ids", required = false)
```

Comment:
It will be more user friendly if we provide a selection list here for users to choose from. There's an example in https://github.com/liferay/liferay-portal/blob/master/modules/apps/account/account-service/src/main/java/com/liferay/account/internal/configuration/AccountEntryGroupConfiguration.java#L33-L34

UI: Site Settings > Accounts
<img width="897" alt="Screenshot 2025-05-07 at 3 03 48 AM" src="https://github.com/user-attachments/assets/bb2118e1-7e2e-4107-a684-fbe0e5966f33" />

Reply:
There will be a UI created in a different story so we don't have to worry about the current one.